department-of-veterans-affairs/caseflow
# Removal of VHA CAMO's "Send to Board Intake" Task Action

Resolves [APPEALS-9093](https://vajira.max.gov/browse/APPEALS-9093)

### Description
Remove VHA CAMO 'Send to Board Intake' task action

### Acceptance Criteria
- [ ] Removal of task action
- [ ] Removal of modal and associated frontend elements
- [ ] Removal of any unnecessary routing to modal in frontend code
- [ ] Removal of any references to the task action/modal in any tests
- [ ] Removal of any orphaned constants like strings that are no longer going to be in use, if any.


### Testing Plan
1. To generate test data
    1.  Open a rails console (`rails c`)
    2. Generate some cases: `10.times.map { FactoryBot.create(:vha_document_search_task) }`
    3.  Make sure they're properly assigned: `PreDocketTask.all.each { |t| t.update!(assigned_to: BvaIntake.singleton) }`
4.  Log in as CAMOADMIN
6. Navigate to the VHA CAMO queue: http://localhost:3000/organizations/vha-camo
7. Click on a case
8. There should not be a task action called 'Send to Board Intake' in the dropdown

<img width="1370" alt="image" src="https://user-images.githubusercontent.com/17840614/222487108-d8c0638b-7d39-46a3-bc17-4c6ae81bbd1a.png">
